DorisProject/src/app/pages/add-category/add-category.component.ts

```diff
@@ -13,9 +16,15 @@ export class AddCategoryComponent implements OnInit {
   addCategoryForm: FormGroup;
   mensaje: String;
   mostrar: boolean;
-  mostrarError:boolean;
+  mostrarError: boolean;
+  responseModal: any;
 
-  constructor(private router: Router, private categoryService: CategoriasService, private formBuilder: FormBuilder) {
+  constructor(private router: Router,
```

Comment:
Codacy has a fix for the issue: [Insert `⏎····`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9258681)
```suggestion
  constructor(
```